basin3d/plugins/usgs.py

```diff
@@ -258,14 +275,14 @@ def _load_point_obj(datasource, json_obj, feature_observed_properties, synthesis
             )],
             # geographical_group_id=huc_accounting_unit_id,
             # geographical_group_type=FeatureTypeEnum.REGION,
-            observed_property_variables=observed_property_variables,
+            observed_property_variables=mf_opv,
             coordinates=Coordinate(
                 absolute=AbsoluteCoordinate(
                     horizontal_position=GeographicCoordinate(
                         **{"latitude": lat,
-                           "longitude": lon,
-                           "datum": json_obj['dec_coord_datum_cd'],
-                           "units": GeographicCoordinate.UNITS_DEC_DEGREES}))
+                            "longitude": lon,
+                            "datum": json_obj['dec_coord_datum_cd'],
+                            "units": GeographicCoordinate.UNITS_DEC_DEGREES}))
```

Comment:
Why are these extra spaces added at the beginning of the line?

Reply:
@dschristianson when I press Cmd+Shift+L, PyCharm reformats/rearranges the code styling.